crates/ibc/Cargo.toml

```diff
@@ -50,6 +50,9 @@ upgrade_client = []
 mocks = ["tendermint-testgen", "tendermint/clock", "cfg-if", "parking_lot"]
 mocks-no-std = ["cfg-if"]
 
+# This feature grants access to additional `validate` and `execute` entrypoints.
+val-exec-entry = []
```

Comment:
Why have a feature? I find it has many downsides
1. it's a crutch against proper documentation. All this does currently is effectively "hide" 2 functions, with no binary size improvements that conditional compilation is typically used for. Don't you think there's a way to properly document these functions such that it's clear which one a given user should use? 
2. Our docs completely hide them from the documentation. This makes discoverability of these entrypoints more difficult. Some projects have a "X feature flag required" notes in their docs, but we don't (and I currently don't know how to do it).